File: python/TimeConversion.py
#!/bin/python3

import os
import sys

def timeConversion(t):
    h, m, s = t[:-2].split(":")
    if t[-2:] == "PM" and h != "12":
        h = int(h) + 12
    elif t[-2:] == "AM" and h == "12":
        h = "00"
    return (str(h)+":"+m+":"+s)

if __name__ == '__main__':
    f = open(os.environ['OUTPUT_PATH'], 'w')

    t = input()

    result = timeConversion(t)

    f.write(result + '\n')

    f.close()

File: scala/TimeConversion.scala
import java.io.PrintWriter
import java.text.SimpleDateFormat

object Solution {

    /*
     * Complete the timeConversion function below.
     */
    def timeConversion(s: String): String = {
        val inputTimeFormat = new SimpleDateFormat("hh:mm:ssa")
        val timeWithDateFormat = inputTimeFormat.parse(s)
        val outputTimeFormat = new SimpleDateFormat("HH:mm:ss")
        val outputTime = outputTimeFormat.format(timeWithDateFormat)

        return outputTime
    }

    def main(args: Array[String]) {
        val scan = scala.io.StdIn

        val fw = new PrintWriter(sys.env("OUTPUT_PATH"))

        val s = scan.readLine

        val result = timeConversion(s)

        fw.println(result)

        fw.close()
    }
}
